clients/orderbook/src/worker.rs

```diff
@@ -335,10 +381,11 @@ where
 				// Commit the state changes in trie
 				trie.commit();
 			},
-			UserActions::Withdraw(withdraw) => self.process_withdraw(withdraw)?,
+			UserActions::Withdraw(withdraw) => self.process_withdraw(withdraw, action.stid)?,
```

Comment:
stid should be renamed to worker_nonce

Reply:
This can be a different issue ( not a priority )

Reply:
Issue [763](https://github.com/Polkadex-Substrate/orderbook/issues/763) will resolve this